app/controllers/concerns/idv/doc_auth_vendor_concern.rb

```diff
@@ -7,16 +7,16 @@ module DocAuthVendorConcern
     # @returns[String] String identifying the vendor to use for doc auth.
     def doc_auth_vendor
       if resolved_authn_context_result.facial_match? || socure_user_set.maxed_users?
-        if doc_auth_vendor_enabled?(Idp::Constants::Vendors::LEXIS_NEXIS)
-          bucket = :lexis_nexis
-        elsif doc_auth_vendor_enabled?(Idp::Constants::Vendors::MOCK)
-          bucket = :mock
-        else
-          return nil
-        end
+        bucket = choose_non_socure_bucket
       else
         bucket = ab_test_bucket(:DOC_AUTH_VENDOR)
       end
+
+      if bucket == :socure
+        if !add_user_to_socure_set
+          bucket = choose_non_socure_bucket # force to lexis_nexis if max user reached
+        end
```

Comment:
can we add a spec for this?

Reply:
@AShukla-GSA confirmed tested already 
https://github.com/18F/identity-idp/blob/2c6ae7c9275d60be132512591f621e629de9642f/spec/controllers/idv/document_capture_controller_spec.rb#L213
https://github.com/18F/identity-idp/blob/2c6ae7c9275d60be132512591f621e629de9642f/spec/controllers/idv/document_capture_controller_spec.rb#L226